app/views/layouts/hero.html.slim

```diff
@@ -14,18 +14,19 @@ html.govuk-template lang='en'
 
     = navigation
 
-    .dfe-content-page--header class='govuk-!-padding-bottom-4'
-      .dfe-width-container 
-        = render 'layouts/banner'
+    main role='main'
+      .dfe-content-page--header class='govuk-!-padding-bottom-4'
+        .dfe-width-container
+          = render 'layouts/banner'
 
-        .noprint= back_link
+          .noprint= back_link
 
-        = render 'layouts/flash'
+          = render 'layouts/flash'
 
-        = yield :hero
+          = yield :hero
 
-    .dfe-width-container
-      main#main-content.govuk-main-wrapper role='main'
-        = yield
+      .dfe-width-container
+        main#main-content.govuk-main-wrapper role='main'
```

Comment:
Why do we remove `#main-content.govuk-main-wrapper` from the other layout but not this one?

Reply:
I have updated where main is so it matches what is mentioned in the ticket. For this line above I will update it so it keeps the styling (this is used for padding some text here)